README.md

```diff
@@ -154,6 +154,47 @@ yarn run dev
 ```
 If changes aren't detected, you might need to increase `fs.inotify.max_user_watches`. See <https://webpack.js.org/configuration/watch/#not-enough-watchers>.
 
+#### Deploying a Custom Image to an OpenShift Cluster
+
+Once you have made changes locally, these instructions will allow you to push changes to a
+ OpenShift Cluster for others to review.  This involves building a local image, pushing the
+ image to an image registry, then updating the OpenShift cluster to pull the new image.
+
+##### Prerequisites
+
+1. Docker v17.05 or higher
+2. An image registry like [quay.io ](https://quay.io/signin/) or [dockerhub](https://hub.docker.com/)
+
+##### Steps
+1. Create a repository in the image registry of your choice to hold the image
+  ```
+  Login to https://quay.io, and create a a empty public repository 'console'
+  ```
+2. Build Image `docker build -t <your-image-name> [<your-git-url> | <path-to-dockerfile>]`
+  ```
+  docker build -t quay.io/myaccount/console github.com/myaccount/console#latestchanges
+  ...or...
+  docker build -t quay.io/myaccount/console .  (will use local Dockerfile in console dir)
+  ```
+3. Push Image to Image Registry `docker push <your-image-name>`.  Make sure docker is logged into your image registry!
+  ```
+  docker push quay.io/myaccount/console
+  ```
+4. Put the console operator in unmanaged state.  Make sure you `oc login` to remote OpenShift cluster. 
+  ```
+  oc edit consoles.operator.openshift.io cluster
+  ```
+  ```yaml
+      spec:
+          managementState: Unmanaged
+  ```
+5. Update the console Deployment with the new image:
+  ```
+  oc set image deploy console console=quay.io/myaccount/console -n openshift-console
+  ```
+Wait for Pods to pull new image & rebuild
+You should now be able to see your development changes on the remote OpenShift cluster!
```

Comment:
I'd add something about how to restore the console to a managed state when done.